ChangeLog

```diff
@@ -41,6 +41,7 @@ ver. 0.9.2 (2014/XX/XXX) - wanna-be-released
    - Monit config for fail2ban in /files/monit
    - New actions:
      - action.d/firewallcmd-multiport and action.d/firewallcmd-allports Thanks Donald Yandt
+     - action.d/nsupdate Thanks Andrew St. Jean
```

Comment:
please describe here very briefly what is this for since "ns" is a loaded acronym ;-)